pom.xml

```
@@ -33,12 +33,12 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/email-ext-plugin</gitHubRepo>
         <java.level>8</java.level>
-        <jenkins.version>2.164.3</jenkins.version>
+        <jenkins.version>2.222.4</jenkins.version>
```

FWIW https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ would recommend
```suggestion
        <jenkins.version>2.222.1</jenkins.version>
```
(I have reservations about that recommendation.)

---

This needs to be 2.222.4 because a transitive dependency (`org.jenkins-ci.plugins:credentials:jar:2.3.13`) requires Jenkins 2.222.4 or higher.

---

Right, the problem to my mind is lack of consistency in the ecosystem. You can feel free to backdate the version in `credentials` and it should be possible to get a quick release I uess.